libs/perun/services/src/lib/dynamicDataSource.ts

```diff
@@ -84,6 +86,48 @@ export class DynamicDataSource<T> implements DataSource<T> {
       });
   }
 
+  getAllMembers(
+    voId: number,
+    attrNames: string[],
+    sortOrder: SortingOrder,
+    totalCount: number,
+    sortColumn: MembersOrderColumn,
+    statuses: VoMemberStatuses[],
+    searchString?: string,
+    groupId?: number,
+    groupStatuses?: MemberGroupStatus[]
+  ): Observable<RichMember[]> {
+    return new Observable((subscriber) => {
+      const requests = [];
+      for (let pageNumber = 0; pageNumber < Math.ceil(totalCount / this.step); pageNumber++) {
+        requests.push(
+          this.dynamicPaginatingService.getMembers(
+            voId,
+            attrNames,
+            sortOrder,
+            pageNumber,
+            this.step,
+            sortColumn,
+            statuses,
+            searchString,
+            groupId,
+            groupStatuses
+          )
+        );
+      }
+      forkJoin(requests).subscribe(
+        (results) => {
+          const mergedData = [].concat(
+            ...results.map((result) => (result as PaginatedRichMembers).data)
+          );
+          subscriber.next(mergedData as RichMember[]);
+          subscriber.complete();
+        },
+        (error) => subscriber.error(error)
+      );
```

Comment:
Use an observer argument if you have more than one argument in the subscribe method (for example, next and error). I know that we have this deprecated usage in many places in our code, but we want to fix it gradually over time. Fix it at all places in this file, please.
```suggestion
      forkJoin(requests).subscribe({
        next: (results) => {
          const mergedData = [].concat(
            ...results.map((result) => (result as PaginatedRichMembers).data)
          );
          subscriber.next(mergedData as RichMember[]);
          subscriber.complete();
        },
        error: (error) => subscriber.error(error)
      });
```